src/EnergyPlus/DisplacementVentMgr.cc

```diff
@@ -596,13 +596,13 @@ void CalcUCSDDV(EnergyPlusData &state, int const ZoneNum) // Which Zonenum
         }
     }
 
-    auto &Zone(state.dataHeatBal->Zone);
+    auto &zone = state.dataHeatBal->Zone(ZoneNum);
```

Comment:
This one was tricky.

Reply:
Because of the variable name?  If so I would try to use IDE tools to change the name to something like XYZ first, then it will be easier to reliably change the reference.